lib/datadog/appsec/extensions.rb

```diff
@@ -30,9 +30,9 @@ def initialize(settings)
 
         # Writer methods
 
-        def instrument(name, options = {})
+        def instrument(name, _options = {})
```

Comment:
I left the `_options` because this is a public API. I do not want to break the contract 😄

Reply:
Should we perhaps just go with `_unused`?

Reply:
Either is OK -- to be honest my comment was more towards not having the `= {}` but I realize now that wasn't particularly clear ;)